conf/portal.application.conf

```diff
@@ -33,6 +33,10 @@ play.modules.enabled += "global.MainModule"
 play.http.actionCreator = "global.ActionCreator"
 play.http.errorHandler = "global.ErrorHandler"
 
+# Http Server
+# ~~~~~
+play.server.http.port = 8080
```

Comment:
Sure I can pull this out. But we need to do this. Port 9000 is a terrible choice ;)